remixjs-uploader-and-player/README.md

```diff
@@ -0,0 +1,74 @@
+# Mux Video
+
+> This example will be removed from this repository once it is accepted and merged into the official Remix examples: [remix-run/examples#441](https://github.com/remix-run/examples/pull/441).
+
+This example features video uploading and playback in a Remix.js application.
+
+This example is useful if you want to build a platform that supports user-uploaded videos. For example:
+
+- Enabling user profile videos
+- Accepting videos for a video contest promotion
+- Allowing customers to upload screencasts that help with troubleshooting a bug
+- Or even the next Youtube, TikTok, or Instagram
+
+## Preview
+
+Open this example on [CodeSandbox](https://codesandbox.com):
+
+[![Open in CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/remix-run/examples/tree/main/mux-video)
+
+## How to use
+
+### Step 1. Create a Remix app with this example
+
+```bash
+npx create-remix@latest --template examples/mux-video
```

Comment:
Will this work if the external PR is not accepted?

Reply:
good callout. gotta fix that.